guides/common/modules/proc_preparing-for-disaster-recovery-with-active-and-passive-project-server-and-backup-and-restore.adoc

```diff
@@ -0,0 +1,48 @@
+[id="preparing-for-disaster-recovery-with-active-and-passive-project-server-and-backup-and-restore"]
+= Preparing for disaster recovery with active and passive {ProjectServer} and backup and restore
+
+Create your passive {ProjectServer} by restoring a backup of your active {ProjectServer}.
+Configure periodic backups of the active server.
+
+.Procedure
+. Define a schedule for periodic offline backups of your active {ProjectServer}.
+Consider your tolerance for potential data loss and your storage options: Taking backups frequently will result in smaller amounts of data loss in case of a disaster, but backups require a significant amount of storage space.
+ifdef::katello,orcharhino,satellite[]
+For information about the size of {Project} backups, see xref:Estimating_the_Size_of_a_Backup_admin[].
+endif::[]
++
+You can combine full backups with incremental backups.
+For an example of a `cron` job that ensures regular backups, see xref:Example_of_a_Weekly_Full_Backup_Followed_by_Daily_Incremental_Backups_{context}[].
+. Schedule periodic offline backups of your active {ProjectServer} to be taken according to the schedule you defined.
+For information about performing backups, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. {Project} stores the backups in the `_/var/{project-context}-backup_` directory by default.
+Ensure that these backup directories are encrypted and regularly synchronized to a secure location.
```

Comment:
```suggestion
. Ensure that the backup directories are encrypted and regularly synchronized to a secure location.
By default, {Project} stores the backups in the `_/var/{project-context}-backup_` directory.
```

suggestion (non-blocking): flip order

You could flip the order to show the instruction first and the explanation second. Your call.